src/utils/withRTLExtension.js

```diff
@@ -10,15 +11,16 @@ const styleDefRegex = /\.[^{}]+\{[^{}]+\}/g;
  * handler adds that extra style definition to the <style /> tag created by Aphrodite.
  */
 function directionSelectorHandler(selector, baseSelector, generateSubtreeStyles) {
-  if (selector !== RTL_SELECTOR) {
+  if (selector !== RTL_SELECTOR && selector !== LTR_SELECTOR) {
     return null;
   }
 
   const generated = generateSubtreeStyles(baseSelector);
+  const prefix = selector === RTL_SELECTOR ? '[dir="rtl"]' : '[dir="ltr"]';
 
   // generated may include more than one style definition (pseudoselectors, matchmedia, etc.).
   // We want to prefix each one individually.
-  return generated.replace(styleDefRegex, g => `[dir="rtl"] ${g}`);
+  return generated.replace(styleDefRegex, g => `${prefix} ${g}`);
```

Comment:
I think if you used a capture group, you wouldn't need the function version of replace here, which might be a perf boost.

```js
const styleDefRegex = /(\.[^{}]+\{[^{}]+\})/g;
...
return generated.replace(styleDefRegex, `${prefix} $1`);
```

Reply:
```
return generated.replace(styleDefRegex, `${prefix} $1`);
```
does not appear to work (at least in my tests) and I have no idea why.

Reply:
Are you sure you added the capture group to the regex? That's pretty key for this change. I'm happy to look at it with you in person.

Reply:
I think I missed the difference in the regex. It works now.